webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -0,0 +1,46 @@
+import * as ToastActions from './react_app/redux/actions/toasts';
+import store from '../javascripts/react_app/redux';
+import $ from 'jquery';
+
+export function notify(message, type, sticky) {
```

Comment:
I do wonder if we need to expose a better interface here for katello, or for "current" foreman ui to be able to provide more attributes (e.g. actions) into the toast.